{"name": "test_upf_session_rule", "status": "broken", "statusDetails": {"message": "configparser.NoSectionError: No section: 'global'", "trace": "self = <configparser.ConfigParser object at 0x7f9c05c94b00>, section = 'global', vars = None\n\n    def _unify_values(self, section, vars):\n        \"\"\"Create a sequence of lookups with 'vars' taking priority over\n        the 'section' which takes priority over the DEFAULTSECT.\n    \n        \"\"\"\n        sectiondict = {}\n        try:\n>           sectiondict = self._sections[section]\nE           KeyError: 'global'\n\n/usr/lib/python3.6/configparser.py:1138: KeyError\n\nDuring handling of the above exception, another exception occurred:\n\n    @pytest.fixture(scope = 'module')\n    def get_test_id():\n>       test_id = Landslide.case_start(get_parameter('case_001'))\n\n../overall/test_003_smf_rel_pdu.py:18: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n../overall/config.py:15: in get_parameter\n    parameter = config.get('global',parameter)\n/usr/lib/python3.6/configparser.py:781: in get\n    d = self._unify_values(section, vars)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <configparser.ConfigParser object at 0x7f9c05c94b00>, section = 'global', vars = None\n\n    def _unify_values(self, section, vars):\n        \"\"\"Create a sequence of lookups with 'vars' taking priority over\n        the 'section' which takes priority over the DEFAULTSECT.\n    \n        \"\"\"\n        sectiondict = {}\n        try:\n            sectiondict = self._sections[section]\n        except KeyError:\n            if section != self.default_section:\n>               raise NoSectionError(section)\nE               configparser.NoSectionError: No section: 'global'\n\n/usr/lib/python3.6/configparser.py:1141: NoSectionError"}, "start": 1566890634601, "stop": 1566890634601, "uuid": "a08c59da-eb1a-42c1-abb5-b00a5fcf415e", "historyId": "cc541b7c1dfe4b1af9a3168b0cdf08a0", "testCaseId": "4addca8e2fcd688787e11500d8832733", "fullName": "test_003_smf_rel_pdu#test_upf_session_rule", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_003_smf_rel_pdu"}, {"name": "subSuite"}, {"name": "host", "value": "ubuntu18"}, {"name": "thread", "value": "5084-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_003_smf_rel_pdu"}]}